service/test/integration2_test.go

```diff
@@ -74,6 +75,9 @@ func startServer(name string, buildFlags protest.BuildFlags, t *testing.T) (clie
 		Debugger: debugger.Config{
 			Backend:        testBackend,
 			CheckGoVersion: true,
+			Packages:       []string{fixture.Source},
+			BuildFlags:     "", // build flags can be an empty string here because the only test that uses it, does not set special flags.
+			Kind:           debugger.ExecutingGeneratedTest,
```

Comment:
This should say ExecutingGeneratedFile, it doesn't really matter but for consistency that's what they are.

Reply:
Makes sense. I changed it.